tocs/4.FAQ/FAQ.md

```diff
@@ -102,3 +102,11 @@ Please load DPU accelerate application. Double check accelerate application list
 ```
 xmutil loadapp <dpu accelerate application name>
 ```
+
+# At first boot into system, what's the account name? password?
+Account name is `petalinux`. And you can set a new password at first time.
+
+# How to access USB?
+You can use command `sudo su` to switch into the `root mode` to access USB.
```

Comment:
You can use the command `sudo su` to switch mode to root for accessing USB.

Reply:
got it